examples/applications/object_detection/retina_net/basic/pascal_voc/train.py

```diff
@@ -0,0 +1,341 @@
+"""
+Title: Train an Object Detection Model on Pascal VOC 2007 using KerasCV
+Author: [lukewood](https://lukewood.xyz)
+Date created: 2022/08/02
+Last modified: 2022/08/02
+Description: Use KerasCV to train a RetinaNet on Pascal VOC 2007.
+"""
+
+"""
+## Overview
+
+KerasCV offers a complete set of APIs to allow you to train your own state-of-the-art,
+production-grade object detection model.  These APIs include object detection specific
+data augmentation techniques, models, and COCO metrics.
+
+To get started, let's sort out all of our imports and define global configuration parameters.
+To support long training runs, experiment tracking via [Weights & Biases](https://wandb.ai),
+and dynamic batch sizing, we define our configuration parameters as command line flags.
+"""
+
+import sys
+
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import wandb
+from absl import flags
+from tensorflow import keras
+from tensorflow.keras import callbacks as callbacks_lib
+from tensorflow.keras import optimizers
+
+import keras_cv
+from keras_cv import bounding_box
+
+flags.DEFINE_boolean("wandb", False, "Whether or not to use wandb.")
+flags.DEFINE_integer("batch_size", 8, "Training and eval batch size.")
+flags.DEFINE_integer("epochs", 1, "Number of training epochs.")
+flags.DEFINE_string("wandb_entity", "keras-team-testing", "wandb entity to use.")
```

Comment:
Consider making this default to none and dropping the `wandb` flag.

Then we can use FLAGS.wandb_entity is not None on line 43, and we don't make users accidentally try to use our WandB entity.

Reply:
Great idea, done.